packages/icons/scripts/copy-and-optimize-to-dist.js

```diff
@@ -0,0 +1,31 @@
+const chalk = require("chalk");
```

Comment:
You could rename this for `optimize-svgs`. Since you optimize the SVG you have to copy the output somewhere it's a consequence of optimisation and doesn't really have to be explicitly specified

Reply:
To be consistant with the other scripts, this script should probably use "meow"